src/BarChart/BarChartHorizontal.story.tsx

```diff
@@ -64,6 +64,54 @@ export const Simple = () => (
   />
 );
 
+export const BarTargetMarker = () => {
+  const sampleData = [
+    {
+      key: 'DLP',
+      data: 10,
+      target: 15
```

Comment:
Not a fan of this API. It doesn't follow the other patterns. I was thinking something could be like: `[10, 15]`. I'd rather extend `data` than add another property.

Reply:
But that type of data used for other things, like that

![Screenshot 2024-11-14 at 16 28 00](https://github.com/user-attachments/assets/9704de18-b1c5-4370-be4b-a76c5695c0fd)

Also that type of data `[10, 15]` will be not clear in "target" context, it's more like period value

Reply:
Ah yes, I forgot about this. Can you look at how other chart libraries do this?